scripts/run_mypy_checks.py

```diff
@@ -139,12 +148,24 @@ def install_mypy_prerequisites(install_globally: bool) -> Tuple[int, str]:
             PYTHON3_CMD, '-m', 'pip', 'install', '-r',
             MYPY_REQUIREMENTS_FILE_PATH
         ]
+        if feconf.OPPIA_IS_DOCKERIZED:
+            cmd = [
+                'pip', 'install', '-r',
+                MYPY_REQUIREMENTS_FILE_PATH, '--find-links=/root/.cache',
+                '--no-index'
+            ]
```

Comment:
I am here trying to maintain the behaviour of the script what it is performing, but yes, this should be removed and the tests can be simply executed with the direct command from the makefile, maybe later after in the future enhancements

Reply:
Have we run this step when building dev-server image? If yes, then why do we need it to be re-run here? Can we remove it? (Maybe pass --skip_install flag when running mypy checks from entrypoint.)

Reply:
> Have we run this step when building dev-server image

yes, in the build step because we have added --no-index flag for making the setup to run in offline mode. so this needed to be downloaded in the `/root/.cache` dir.

> Can we remove it?

I feel to optimise all the scripts for the tests outside the Docker project (as the future enhancements with the high priority). would that be alright?

Reply:
If I'm understanding it correctly, we already have this step preformed in the Dockerfile as you are saying. SO why we need to rerun this again here in docker container?

Reply:
as per our offline discussion, removed it and changed the script accordingly to not to install packages under docker env